package.json

```diff
@@ -135,6 +135,7 @@
   "dependencies": {
     "@babel/runtime-corejs3": "^7.4.3",
     "blueimp-load-image": "~2.12.2",
+    "chromedriver": "^78.0.1",
```

Comment:
This dependency has nothing to do with this PR, right?

Also shouldn't it be a devDependency?

Reply:
Hmmm....I think this got introduced when I was trying to get the tests working & @bartzet mentioned running `npm install chromedriver` but odd that other PRs are working fine & don't seem to have this in their `package.json`. Should be in devDependency either way. Good spot.